### ISSUE TITLE
Stop Working

### ISSUE THREAD

**Issue by OLH21** [dhamaniasad/obsidian-rich-links#27]
Was ok, it has stop generating link, old links show ok, and seems to work because it shows an error message if you do not have a link selected. disabled all other plugin and theme and it's still the same. Maybe it's the latest obsidian update :)

**Comment by dhamaniasad** [comment]
Should be working now

**Comment by OLH21** [comment]
Can we have a hint on what was the problem ? is ther and external server call ? thx anyway 
BTW it has never been working with all link some refuse to work whatever what's the reason ?

**Comment by dhamaniasad** [comment]
Yes there's an external server call, and that server had been shut down. It uses opengraph or meta tags to fetch the link description, if those are missing it'll fail to work. Can you share an example link it didn't work?

The reason we need the server is because of CORS, where servers block requests for security, so if I'm on hackersite.com it's not able to make requests to google.com on my behalf. Obsidian runs in a browser environment where CORS is respected and on servers it's not. So without a server a lot of links would fail to work.

**Comment by kaykhan** [comment]
Has this stopped working again, when i click the ribbon it just says "Select a URL to convert to a rich link" but ive already highlighted the link?

**Comment by OLH21** [comment]
Stopped working on my side too ( i have a waiting message only) , don't know when it begins since it's been a while i havent used it
